tests/ctst/steps/quotas/quotas.ts

```diff
@@ -88,15 +81,40 @@ Before({ tags: '@Quotas', timeout: 1200000 }, async function ({ gherkinDocument,
 
     const configuration: typeof output = JSON.parse(fs.readFileSync(`/tmp/${featureName}`, 'utf8')) as typeof output;
     const key = hashStringAndKeepFirst20Characters(`${pickle.astNodeIds[1]}`);
-    world.parameters.logger?.debug('Scenario key', { key, from: `${pickle.astNodeIds[1]}`, configuration });
-    const config = configuration[key];
-    world.resetGlobalType();
+    world.logger.debug('Scenario key', { key, from: `${pickle.astNodeIds[1]}`, configuration });
     // Save the bucket name for the scenario
     world.addToSaved('bucketName', key);
-    // Save the account name for the scenario
-    Zenko.saveAccountAccessKeys(config.AccessKey, config.SecretKey);
     world.addToSaved('accountName', key);
-});
+    // Save the account name for the scenario
+    Identity.addIdentity(IdentityEnum.ACCOUNT, key, configuration[key], undefined, true, true);
+}
+
+export async function quotaScenarioteardown(world: Zenko) {
```

Comment:
nit: `tearndownQuotaScenario`

Reply:
I will do it in the next PR (for kafkacleaner flakiness & azure locations), thanks